src/modules/accommodations/entities/accommodation.entity.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  AccommodationCategory,
+  AccommodationStatus,
+} from 'src/shared/constants';
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  OneToMany,
+  PrimaryGeneratedColumn,
+  UpdateDateColumn,
+} from 'typeorm';
+import { Amenity } from './amenity.entity';
+
+@Entity()
+export class Accommodation {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column({
+    type: 'varchar',
+    length: 125,
+  })
+  name: string;
+
+  @Column({
+    type: 'varchar',
+    length: 255,
+  })
+  description: string;
+
+  @Column({
+    type: 'varchar',
+    length: 200,
+  })
+  location: string;
+
+  @Column({
+    name: 'price_per_night',
+    type: 'decimal',
+  })
+  price_per_night: number;
+
+  @Column({
+    type: 'enum',
+    enum: AccommodationCategory,
+    array: true,
+    default: [AccommodationCategory.HOTEL],
+  })
+  categories: AccommodationCategory[];
```

Comment:
why is it an array? I think we should keep it simple and have one category. If you want to add multiple categories, we can add tags field instead for this kind of stuff.

Reply:
I think you can also add an index for it if you make the category field singular and not an array. It will be useful because we will be filtering accommodations by category a lot.

Reply:
Ok, I'll do it